### DISCUSSION TITLE
Check if item is available to ship to postal code

### DISCUSSION THREAD

**Discussion by greavettey** [Ephigenia/ikea-availability-checker#343] [Q&A]
Thought I'd ask here since I don't see anything in the readme or documentation that suggests this is implemented. Is a feature like this possible with the current API/package? Thanks

**Comment by Ephigenia** [comment]
No there’s nothing in this project nor an API endpoint that I know of that supports this. One could combine a database with zip codes and their locations/areas and calculate distances using it.
